CONTRIBUTING.md

```diff
@@ -1,52 +1,72 @@
-# Contributing to Icon Editor for LabVIEW
-
-Hello there! So glad you want to help out with the LabVIEW Icon Editor 🤗 You’re the best. Here’s a guide for how you can contribute to the project.
-
 # The Open Source LabVIEW Icon Editor
 
-Just recently, the LabVIEW Icon Editor has been made open source to encourage collaboration between NI and the LabVIEW Community. Now you can find the source code of the LabVIEW Icon Editor in this repository. We are in a state of transition. This is the current structure:
-
-* LabVIEW Icon Editor `labview-icon-editor` version {latest_version} is on branch {branch_name}
-* The latest version of the LabVIEW Icon Editor (not published yet) can be found {latest_version_link}
-
-# Our Contributing Philosophy
+Just recently, the LabVIEW Icon Editor has been made open source to encourage collaboration between NI and the LabVIEW Community. Every new build of LabVIEW will grab the icon editor that has been pushed into the main branch from this repo.
```

Comment:
Use backticks when referencing the `main` branch to avoid any ambiguity that "main" is the name of the branch and not just the default branch. @svelderrainruiz